src/Logic/TimeCode.cs

```diff
@@ -95,8 +93,7 @@ public int Minutes
             }
             set
             {
-                var ts = TimeSpan;
-                TimeSpan = new TimeSpan(0, ts.Hours, value, ts.Seconds, ts.Milliseconds);
+                _totalMilliseconds = TimeSpan.FromMinutes(value).TotalMilliseconds;

```

Comment:
only for "TotalMinutes" (if that was there) - but it's covered by unit tests now, so that's nice :)